src/graphql/apollo/defaults.ts

```diff
@@ -1,10 +0,0 @@
-const defaults = {
-  events: {
-    name: '',
-  },
-  patients: {
-    gender: '',
-  },
-};
-
-export default defaults;
```

Comment:
what was this doing before, do you know?

Reply:
If I recall correctly, this file wasn't being used. I think it used to be used for setting a "default" cache.